hikari/impl/interaction_server.py

```diff
@@ -365,6 +384,18 @@ async def close(self) -> None:
         await self._server.cleanup()
         self._server = None
         self._application_fetch_lock = None
+
+        for task in self._running_generator_listeners:
+            if not task.done() and not task.cancelled():
+                task.cancel()
```

Comment:
This method is documented as a graceful shutdown but this doesn't feel graceful, would it be worth it adding a force argument or changing the docs around this?

Reply:
That is indeed true, didnt even think about it. Will revisit

Thanks for pointing it out!

Reply:
Fixed to be in-line with the way aiohttp handles shutdown